deptofdefense/move.mil
# Locator Maps

## Checklist

I have…


## Summary of Changes

- Enables the user to search for PPPO offices close to either a ZIP code or their current location.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. git checkout `pppo-locator`,
1. set up development dependencies according to [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md),
1. run `bin/rails server`, and
1. load up [http://localhost:3000](http://localhost:3000) in your Web browser of choice.

## Screenshots
